### DISCUSSION TITLE
Estimated allowed amount

### DISCUSSION THREAD

**Discussion by sklukasatphs** [CMSgov/hospital-price-transparency#146] [Q&A] [answered]
most of our contract agreements are for percent of billed charges or base fee schedule multiplied by contracted rate.  **is the actual result** of those calculations what we are expected to fill the estimated allowed amount column?  for instance:
$1,000 charge x 81% of billed charges = $810 in the estimated allowed amount column?
Medicare OPPS x 105% = 10021 FNA  pymt rate $379.92 x 105% = 398.92

thanks

**Comment by RidenourK** [comment, marked as answer]
@ sklukasatphs. Thank you for your question. 

Please refer to GitHub posts #132, and #137.
